scout/server/utils.py

```diff
@@ -15,19 +15,23 @@
 LOG = logging.getLogger(__name__)
 
 
-def html_to_pdf_file(html_string, orientation, dpi=96, margins=["1.5cm", "1cm", "1cm", "1cm"]):
+def html_to_pdf_file(
+    html_string, orientation, dpi=96, margins=["1.5cm", "1cm", "1cm", "1cm"], zoom=1
+):
     """Creates a pdf file from the content of an HTML file
     Args:
         html_string(string): an HTML string to be rendered as PDF
         orientation(string): landscape, portrait
         dpi(int): dot density of the page to be printed
         margins(list): [ margin-top, margin-right, margin-bottom, margin-left], in cm
+        zoom(float): increase of decrease the size of the content on the pages
 
     Returns:
         bytes_file(BytesIO): a BytesIO file
     """
     options = {
         "page-size": "A4",
+        "zoom": zoom,
```

Comment:
This is not fixing the issue of the text outside the page borders, but it is useful when the contents of a page can be made smaller to fit more text in the page